scene/gui/subviewport_container.cpp

```diff
@@ -223,6 +223,22 @@ void SubViewportContainer::unhandled_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+TypedArray<String> SubViewportContainer::get_configuration_warnings() const {
+	TypedArray<String> warnings = Node::get_configuration_warnings();
+
+	bool has_viewport = false;
+	for (int i = 0; i < get_child_count(); i++) {
+		if (get_child(i)->is_class("SubViewport")) {
```

Comment:
```suggestion
		if (Object::cast_to<SubViewport>(get_child(i))) {
```

Reply:
I need to remember that.
Thanks for your comments. Updated the PR to include your suggestions.